lib/find-emails.js

```diff
@@ -75,11 +82,18 @@ module.exports = function findEmails (args, callback) {
     port: args.port,
     tls: true,
     keepAlive: false,
-    // debug: function (info) { console.log(info) },
     tlsOptions: {
       secureProtocol: 'TLSv1_method'
     }
-  })
+  }
+
+  if (args.debug) {
+    imapOptions.debug = function (info) {
```

Comment:
I mean there is a logger function with different levels, eg
```js
log('info', 'my info message')
log('debug', 'my verbose debug msg')
```

Then you can just say something like
```js
imapOptions.debug = log.bind(null, ['debug'])
```